server/api.js

```diff
@@ -19,74 +16,129 @@ const router = new Router();
 // 	});
 // });
 
-router.get("/", (_, res, next) => {
-	const client = getClient();
-	client.connect((err) => {
-		if (err) {
-			return next(err);
-		}
```

Comment:
Hi Mo. Dot you thing it is good to error checking before connecting to the collection? The reason (as I remembered from Jon) is to check if we are connecting to the server!

Reply:
I agree and I tried to implement it, but in the example, above we are connecting the router first so we have "next" to return the error. How would you suggest to handle the error in the new code? @tayfuntayfun

Reply:
It is not meant for error handling but just keeping the less work. However, it is not an important thing.